playbooks/openstack/inventory.py

```diff
@@ -121,11 +121,15 @@ def build_inventory():
     '''Build the dynamic inventory.'''
     cloud = shade.openstack_cloud()
 
+    # Use an environment variable to optionally skip returning the app nodes.
+    no_app_nodes = os.environ.get('INV_NO_APP_NODES', 'false').lower() == "true"
```

Comment:
Maybe it's just me, but I'd understand it better if it wasn't negated. Something like:

    show_compute_nodes = os.environ.get('OPENSTACK_SHOW_COMPUTE_NODES', 'true').lower() == 'true'

and then below, the if would say:

    if 'metadata' in server and 'clusterid' in server.metadata and
    (server.metadata['sub-host-type'] != 'app' or show_compute_nodes)

Reply:
You are right. I will change to what you suggest.